composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "gmponos/guzzle_logger",
+    "name": "covergenius/guzzle_logger",
```

Comment:
Don't think you'll get this merged with references to our own repo still in it ;-)
Then again, there already are #50 and #52 which do the same thing as this PR, will have their 1st birthday in the very near future, and also have not yet been merged.

Don't think this package is still maintained?

Reply:
Sorry I did not mean to open this PR.

We created another repo and will maintain it since they haven't been maintaning it. https://packagist.org/packages/covergenius/guzzle_logger